Topics/Software_Engineering/Retro.md

```diff
@@ -0,0 +1,88 @@
+
+If you have any issues or concerns brought up from previous retros, now is the time to ask the team on whether any potential solutions were thought up of that could help alleviate those issues.
+
+Don't forget to celebrate any successes that occur, no matter how small. It's important for the team to understand what was done that was successful, so that those actions can become reproduceable!
+
+#### Issue Example
+For example, say that PRs tend to take a long while to be reviewed and completed, ask the team if there were any concrete examples of PRs that were up for a long time. Ask them the context behind that PR; Why was is up for so long? Was there someone responsible for reviewing it? Were there issues with that PR? Did team members know that there was a PR?
+
+Have the team come up with quick ideas to solve this problem? Is there a communication problem that needs to be addressed? Is there a problem with the way PRs are being reviewed? Are there not enough people taking time to review the PRs?
+
+The important thing is to look for solutions that work best for your team. There is never a solution that works for every team. For this example, sometimes people need to be randomly assigned to a PR, other times, people need to promise to carve out time to address a PR. Sometimes, it's just an etiquette review to help people figure out the right and wrong way to go about reviewing a PR. Have your team discuss what might be causing an issue.
+
+Make sure that you allow time for problems to be addressed, not a single problem. It is the person leading the retro's responsibility to table any discussions that should be addressed in a separate meeting. Though retros are meant to do so, sometimes problems are large enough to warrant their own separate meeting.
+
+### Wrapping up the meeting
+
+Make sure that you give the team a chance to discuss any problems and to have them aware of them. Record any problems or issues, and their potential solutions. They should be brought up again during the next retro to see if any effort was made to address them. It is a good idea to keep this data in a living document, so that team has access to this information.
+
+## Key Takeaways
+- Retros are a group effort! Everyone should be aware of what happened during the sprint and be willing to problem-solve!
+- Try to host meetings for your retros within a few days after your previous sprint
+- Gather information and important data points, like task completion and absenses
+- Get your team members to summarize what happened over the sprint
+- Log any issues and successful ideas that hindered/aided task completion. Be sure to note down things to consider for the next sprint and have them on hand for the next retro.
+
+## Resources
```

Comment:
As a part of the resources, you could mention and possibly recommend specific digital tools or platforms that facilitate retros, especially for remote or distributed teams like CSC301 teams, which would add practical value. This could include tools for collaborative note-taking, task tracking, or virtual whiteboards. (Google Docs, Zoom whiteboards, etc.)

Reply:
Hmm, I like the idea. I'll add in a few tools, but this is mostly a general project management topic, so I will likely not broach upon the subject too much.